sdv/multi_table/base.py

```diff
@@ -298,7 +298,7 @@ def preprocess(self, data):
         for table_name, table_data in data.items():
             synthesizer = self._table_synthesizers[table_name]
             self._assign_table_transformers(synthesizer, table_name, table_data)
-            processed_data[table_name] = synthesizer.preprocess(table_data)
+            processed_data[table_name] = synthesizer._preprocess(table_data)
```

Comment:
This change was to stop getting already fitted warnings from each table since we already give a warning in `multi_table.preprocessing`.